Lesson32/feyza/my-app/src/shared-components/typography/BodyM/styles.css

```diff
@@ -0,0 +1,9 @@
+.text-m {
+  color: var(--Dark-Navy, #313e51);
+  /* Body M */
+  font-family: Rubik;
+  font-size: 24px;
```

Comment:
Font size in indeed 24 in the example but for bodyM it should be 12px 
![Screenshot 2024-11-18 at 10 19 19](https://github.com/user-attachments/assets/f195ff7e-e855-4be3-abf1-b81d32014802)